packages/extension-ui/src/Popup/Forget.tsx

```diff
@@ -67,7 +68,14 @@ function Forget({
       />
       <span>
         {t<string>('How to restore your account?')}&nbsp;
-        <span className='link'>{` ${t<string>('Learn more')}`}</span>
+        <a
+          className='link'
+          href={LINKS.FORGET}
+          rel='noreferrer'
+          target='_blank'
+        >
+          {t<string>('Learn more')}
+        </a>
```

Comment:
It's used in so many places this probably could be just moved into separate component and used like `<LearnMore href={} />` in all places it's needed? With optional children if you really need to change the copy in some cases.